src/models/orders.rs

```diff
@@ -0,0 +1,142 @@
+extern crate anyhow;
+extern crate ethabi;
+extern crate ethsign;
+extern crate rustc_hex;
+extern crate serde_json;
+extern crate web3;
+
+use anyhow::Result;
+use ethabi::encode;
+use ethsign::Signature;
+use serde::{Deserialize, Serialize};
+use std::cmp::Ord;
+use std::cmp::Ordering;
+use std::cmp::PartialOrd;
+use std::ops::Mul;
```

Comment:
All `std::ops::*` traits are for operator overloading. So `Mul::mul(a, b)` (or `a.mul(b)`) is exactly equivalent to `a * b`. In fact, the `*` operator looks for the `Mul` implementation so that it knows what to call, so in general, you shouldn't need to `use` `std::ops::*` traits unless you intend on implementing them or using them as type constraints for generic methods.